sound/soc/sof/ipc4-topology.c

```diff
@@ -1801,6 +1801,7 @@ static int sof_ipc4_dai_config(struct snd_sof_dev *sdev, struct snd_sof_widget *
 	case SOF_DAI_INTEL_HDA:
 		gtw_attr = ipc4_copier->gtw_attr;
 		gtw_attr->lp_buffer_alloc = pipeline->lp_mode;
+		pipeline->skip_during_fe_trigger = true;
```

Comment:
I am still in the dark on why we need to skip only HDAudio pipelines?

Reply:
Can't we detect this rather than add a flag that duplicates information that can be discovered?

Reply:
Don't we have a snd_sof_dai -> snd_soc_dai helper? I don't really understand why we don't have a 1:1 mapping.

Reply:
we don't have a 1:1 mapping, Pierre. I can add this in a follow up maybe

Reply:
That would be good indeed